packages/blocks/image/src/components/Media.tsx

```diff
@@ -79,12 +79,12 @@ function getLinkGroup(params: {
   return matchingLinkGroup;
 }
 
-function getLinkedEntities<T>(params: {
+function getLinkedEntities(params: {
   sourceEntityId: string;
   path: string;
   linkGroups: BlockProtocolLinkGroup[];
   linkedEntities: BlockProtocolEntity[];
-}): (BlockProtocolEntity & T)[] | null {
+}): (BlockProtocolEntity & { url: string })[] | null {
```

Comment:
Additional change for resolving an old comment: https://github.com/hashintel/dev/pull/447#discussion_r784901100